### PR TITLE
Add Anguils

### PR DESCRIPTION
**Pull request by ignamiranda**
Anguils are electric eel folk

### CODE REVIEW

File: Species/Humanoid/Anguil.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+aliases: Anguils
+---
+# Anguil
+
+Anguils are a unique species of humanoids that possess distinct, eel-like features and abilities. They have elongated bodies similar to that of an eel. They have the ability to produce electric shocks, making them dangerous in battle. These electrifying shocks can stun or incapacitate their enemies, giving the Anguils an advantage in battle.
+
+According to their myths, the gods of water and lightning created the Anguils. Legend has it that the ferocity and adaptability of the eel eel impressed the gods, and decided to imbue a humanoid species with some of these traits. They crafted the Anguils with sleek bodies and powerful muscles, giving them exceptional speed and agility in water. Additionally, they granted them the ability to generate electric shocks, which the Anguils could use for both defense and hunting.
```
Comment (macniel) [inline Species/Humanoid/Anguil.md:8]:
```suggestion
According to their myths, the gods of water and lightning created the Anguils. Legend has it that the ferocity and adaptability of the eel impressed the gods, and decided to imbue a humanoid species with some of these traits. They crafted the Anguils with sleek bodies and powerful muscles, giving them exceptional speed and agility in water. Additionally, they granted them the ability to generate electric shocks, which the Anguils could use for both defense and hunting.
```

**Review by macniel — CHANGES_REQUESTED** [review]
suggestion is that there is a double eel that impressed the gods. Besides that, LGTM.

```suggestion
According to their myths, the gods of water and lightning created the Anguils. Legend has it that the ferocity and adaptability of the eel impressed the gods, and decided to imbue a humanoid species with some of these traits. They crafted the Anguils with sleek bodies and powerful muscles, giving them exceptional speed and agility in water. Additionally, they granted them the ability to generate electric shocks, which the Anguils could use for both defense and hunting.
```

**Comment by ignamiranda** [comment]
Fixed it. One more vote and GitConsensus should merge the change after 12 hours. I don't even know if it's working yet so fingers crossed lol.

**Comment by macniel** [comment]
Oh, so giving approvals on the PR doesnt do anything and the merge process is governed by GitConsensus instead?

**Review by macniel — APPROVED** [review]

**Comment by ignamiranda** [comment]
Correct. I can still merge the request manually but I want to see if the system is working.

**Pull request merged**